website/static/games/ridgerider_games/minesweeper/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Minesweeper
+        </title>
+        <script src="minesweeper.js"></script>
+        <link rel = "stylesheet" href="minesweeper.css"></link>
+    </head>
```

Comment:
![shop](https://user-images.githubusercontent.com/77545656/157785054-9462f767-2f9a-4423-8e4b-3ee918861567.png)
this is trash. dont use head. bad practice. denied